### DISCUSSION TITLE
Reset css?

### DISCUSSION THREAD

**Discussion by Rechdan** [microsoft/fluentui#28632] [Q&A]
The `<html>` and `<body> ` of my webpage are still with basic styling, how do I reset that with Fluent UI?

**Comment by TeshaneCrawford** [comment]
In the Fluent UI v9 documentation there's a new Api from the Griffel CSS-in-JS for react called makeResetStyles that you can use to reset browser styles. 

https://griffel.js.org/react/api/make-reset-styles

At times I just use a regular CSS reset stylings to fix that issue
